_data/toc.yaml

```diff
@@ -1386,6 +1386,12 @@ manuals:
             title: Recover your Docker Hub account
           - path: /docker-hub/2fa/new-recovery-code/
             title: Generate a new recovery code
+      - sectiontitle: Single Sign-on
+        section:
+          -  path: /single-sign-on/
+             title: Single-Sign-on
+          -  path: /single-sign-on/faqs
```

Comment:
As the title of this section is also Single Sign-on, can we call this Configure SSO?

Reply:
![image](https://user-images.githubusercontent.com/47779042/146060484-2f11d868-c598-45d4-8d42-db53c0b03f42.png)

Reply:
Good catch! Yes, updating it now @usha-mandya